common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -508,3 +508,65 @@ def test_studio_settings_page_a11y(self):
         })
 
         self.settings_page.a11y_audit.check_for_accessibility_errors()
+
+
+class AdvancedSettingsTest(StudioCourseTest):
+    """
+    Tests for Advanced Settings tab in studio.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(AdvancedSettingsTest, self).setUp()
+        self.advanced_settings = AdvancedSettingsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        self.type_fields = ['Course Display Name', 'Advanced Module List', 'Discussion Topic Mapping',
+                            'Maximum Attempts', 'Course Announcement Date']
+
+        # Before every test, make sure to visit the page first
+        self.advanced_settings.visit()
+        self.assertTrue(self.advanced_settings.is_browser_on_page())
+
+    def test_cancel_edit_key_value(self):
+        """
+        Scenario: Test cancel editing key value
+        Given I am on the Advanced Course Settings page in Studio
+        When I edit the value of a policy key
+        And I press the "Cancel" notification button
+        Then the policy key value is unchanged
+        And I reload the page
+        Then the policy key value is unchanged
+        """
+        default_course_name = self.advanced_settings.get('Course Display Name')
+        # Change the value of 'Course Display Name'.
+        self.advanced_settings.set_value_without_saving('Course Display Name', 1)
+        # Cancel and don't save the value when asked in notification.
+        self.advanced_settings.cancel()
+        self.assertEquals(self.advanced_settings.get('Course Display Name'), default_course_name)
+        # Refresh the page
+        self.advanced_settings.refresh_and_wait_for_load()
+        # 'Course Display Name' should be same as before.
+        self.assertEquals(self.advanced_settings.get('Course Display Name'), default_course_name)
+
+    def test_edit_key_value(self):
+        """
+        Scenario: Test editing key value
+        Given I am on the Advanced Course Settings page in Studio
+        When I edit the value of a policy key and save
+        Then the policy key value is changed
+        And I reload the page
+        Then the policy key value is changed
+        """
+        course_name_to_set = "New Course Name"
```

Comment:
Is the issue that the value returned by self.advanced_settings.get('Course Display Name') has double quotes at the start and end of the string?

If this is the case, you could use that as your expected value.
I.e. (hard to see, but '" is a single then a double quote, and "' is a double then a single quote)
```
expected_name = '"Course Display Name"'
self.assertEquals(self.advanced_settings.get('Course Display Name'), expected_name)
```